sssd_test_framework/roles/ipa.py

```diff
@@ -282,6 +282,9 @@ def test_example(client: Client, ipa: IPA):
         """
         return IPASudoRule(self, name)
 
+    def idview(self, name: str) -> IPAIdView:
```

Comment:
Can you please add a working example of idviews, use any of the functions as an example.

Reply:
Added, please check.